src/crimson/common/errorator.h

```diff
@@ -352,6 +352,14 @@ struct errorator {
     template <class ErrorVisitor, class Futurator>
     friend class maybe_handle_error_t;
 
+    // any `seastar::futurize` specialization must be able to access the base.
+    // see : `satisfy_with_result_of()` far below.
+    template <typename>
+    friend class seastar::futurize;
```

Comment:
No need to `plainify()` an errorated `future` nor touch the `seastar::future`. I think I missed that initially.